.github/workflows/wheels-publish.yaml

```diff
@@ -0,0 +1,75 @@
+name: Publish RAPIDS wheels
+
+on:
+  workflow_call:
+    inputs:
+      # repo and branch
+      repo:
+        type: string
+      branch:
+        type: string
+      date:
+        type: string
+      sha:
+        type: string
+      build_type:
+        required: true
+        type: string
+
+      # general settings
+      package-name:
+        required: true
+        type: string
+
+permissions:
+  actions: read
+  checks: none
+  contents: read
+  deployments: none
+  discussions: none
+  id-token: write
+  issues: none
+  packages: read
+  pages: none
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+jobs:
+  wheel-publish:
+    name: wheels publish
+    runs-on: ubuntu-latest
+    container:
+      # ctk version of the container is irrelevant in the publish step
+      # it's simply a launcher for twine
+      image: "rapidsai/manylinux_v2_2014:cuda-devel-11.8.0-centos7"
```

Comment:
Should we change this image?

Reply:
Good eye. Switching to `rapidsai/ci-wheel:cuda12.0.1-centos7-py3.10`